src/Tools/dotnet-monitor/Exceptions/ExceptionsService.cs

```diff
@@ -48,6 +55,22 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                 {
                     // Get default process
                     IProcessInfo pi = await _diagnosticServices.GetProcessAsync(processKey: null, stoppingToken);
+
+                    // If previous checked configuration and it did not pass, do not check again or attempt
+                    // to start the event pipe session.
+                    UniqueProcessKey key = new(pi.EndpointInfo.ProcessId, pi.EndpointInfo.RuntimeInstanceCookie);
+                    if (_unconfiguredProcesses.Contains(key))
+                    {
+                        throw new MonitoringException(string.Empty);
```

Comment:
Exception message?

Reply:
I'll switch this to a non-MonitoringException since it is not intended to be observable by the user and add a comment to that effect.